app/forms/provider_interface/provider_user_invitation_wizard.rb

```diff
@@ -177,5 +141,17 @@ def next_provider_needing_permissions_setup
     def any_provider_needs_permissions_setup?
       next_provider_needing_permissions_setup.present?
     end
+
+    def sanitize_attributes(params)
+      delete_permissions_if_view_applications_only(params)
+    end
+
+    def update_review_mode
```

Comment:
I find this a bit confusing — why does it do one thing in the module and the opposite out here?

Reply:
This is how the provider_user_invitaiton_form behaves and I'm not entirely sure why it's true in one instance and not in the other, which is why I also extracted this out of the initializer to enable developers to configure it as they need to. Maybe something to look at separately?